code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -29,7 +30,13 @@
 	else
 		w_class = ITEM_SIZE_SMALL
 
-/obj/item/weapon/reagent_containers/attackby(obj/item/weapon/W as obj, mob/user as mob)
+/obj/item/weapon/reagent_containers/ivbag/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(istype(W, /obj/item/weapon/pen))
+		written_info = sanitize(input("Notes: ", text("IV Bag")))
+		if(written_info)
+			desc = "Flexible bag for IV injectors.\nA note is scribbled on the bag. \"[written_info]\""
+		else
+			desc = "Flexible bag for IV injectors."
 
```

Comment:
put a return TRUE at the bottom of inside the pen check, and a  .=..() at the bottom of the function outside the pen check